install_script.sh.template

```diff
@@ -87,6 +87,7 @@ function report_telemetry() {
     safe_agent_version=$(echo -n "$agent_major_version" | tr '\n' ' ' | tr '"' '_')
   fi
 
+  APM_TELEMETRY_SAFE_AGENT_VERSION_OVERRIDE_PLACEHOLDER
```

Comment:
IIUC, this is used to set the `agent_version` tag in the telemetry event to `noagent_autoinstrumentation` when using the `install_script_docker_injection.sh` script only. What happens if the `install_script_agent7.sh` script is used, with `DD_NO_AGENT_INSTALL` / `DD_APM_INSTRUMENTATION_ENABLED="docker"` set?

Reply:
With `install_script_agent7.sh` we'll just report the Agent version. This is intentional, because APM want to track how much the individual scripts get used and not so much how exactly they're used (at least for now - this may change in the future).

Even though `install_script_agent7.sh` will still support this case, all our docs will recommend using `install_script_docker_injection.sh` for this case.